### PR TITLE
Guess-Who project - Marielle Ohlsson

### PR DESCRIPTION
**Pull request by Marielle8**
My goal with this project was to fulfill the blue level that technigo had for requirment.

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -244,100 +247,156 @@ const generateBoard = () => {
   })
 }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
+// Selects a person from the characters array and set as the value of the variable called secret
```
Comment (AsaHildebrand) [inline code/script.js:250]: Very good and clear explanations!

**Review by AsaHildebrand — APPROVED** [review]
Good job with the code, I can only find small things that could easily be fixed :) As far as I can see all of the requirements for the blue level and the general requirements are fulfilled. Clear and understandable code with comments that explains very well!

File: code/script.js [hunk 2]
```diff
@@ -244,100 +247,156 @@ const generateBoard = () => {
...
 
-// This function to start (and restart) the game
+// Starts (and restarts) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  winOrLose.style.display = "none";
+  winOrLoseText.innerHTML = "";
+  board.style.display = 'flex';
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/script.js:260]: Remember that well configured VScode menage semicolons for you. go to the settings and search for the semicolons. If you have problems with that check if you have your Prettier extension installed. 🤓

File: code/script.js [hunk 3]
```diff
@@ -244,100 +247,156 @@ const generateBoard = () => {
...
 }
 
-// This function should be invoked when you click on 'Find Out'.
+// Invoked when 'Find Out' is clicked.
 const checkQuestion = () => {
-  // Compare the currentQuestion with the secret person.
-  // See if we should keep or remove people based on that
-  // Then invoke filterCharacters
+  let keep
+  if(currentQuestion.attribute === "hairColor" && currentQuestion.value === secret.hairColor) {
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/script.js:299]: this is also a thing that should be provided by autoformatting options in the VScode 🤓

File: code/script.js [hunk 4]
```diff
@@ -244,100 +247,156 @@ const generateBoard = () => {
...
-  // 1. Check if the suspect is the same as the secret person's name
-  // 2. Set a Message to show in the win or lose section accordingly
-  // 3. Show the win or lose section
-  // 4. Hide the game board
+  if (suspect === secret.name) {
+  winOrLoseText.innerHTML = 'Congratulations, you Won!'
+  } else {
+    winOrLoseText.innerHTML = 'Sorry, you lost'
+  }
+  winOrLose.style.display = "block";
```
Comment (KarolCislowski) [inline code/script.js:390]: If you make it 'flex' it would look nicer IMHO 🎨

**Review by KarolCislowski — APPROVED** [review]
This is a very nice project 🌈 

Remember that you can always come back to it when you have some spare time and add more functionalities to make it an even greater portfolio asset! 🦄

**Pull request closed without merging**